### PR TITLE
Add firewall_prometheus resource for docker_exporter

### PR DESCRIPTION
**Pull request by ramereth**
Related and dependant to osuosl-cookbooks/firewall#143

### CODE REVIEW

**Comment by ramereth** [comment]
[Prometheus firewall refactor](https://trello.com/c/vJIreW2K/1374-prometheus-firewall-refactor)

File: Berksfile [hunk 1]
```diff
@@ -3,6 +3,6 @@ source 'https://supermarket.chef.io'
 solver :ruby, :required
 
 cookbook 'docker_test', path: 'test/cookbooks/docker_test'
-cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
+cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall', branch: 'ramereth/promtheus-resource'
```
Comment (doublej472) [inline Berksfile]: Remember to remove this branch specifier before merging!

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.6.1.  Have a nice day!

**Pull request merged**